source/template.yaml

```diff
@@ -1,615 +0,0 @@
-AWSTemplateFormatVersion: '2010-09-09'
-Transform: AWS::Serverless-2016-10-31
-
-Parameters:
-  gitsha:
-    Type: String
-    Description: Git sha
-  environment:
-    Type: String
-    Description: Deployment environment
-    Default: "dev"
-
-Metadata:
-  AWS::ServerlessRepo::Application:
-    Name: login-idp-functions
-    Description: login.gov idp functions
-    Author: logindotgov
-    SpdxLicenseId: CC0-1.0
-    HomePageUrl: https://github.com/18f/identity-idp-functions
-    SemanticVersion: 0.0.1
-    SourceCodeUrl: https://github.com/18f/identity-idp-functions
-  AWS::CloudFormation::Interface:
-    ParameterGroups:
-      -
-        Label:
-          default: Application Settings
-        Parameters:
-          - gitsha
-          - environment
-
-Description: >
-  identity-idp-functions
-
-  IDP functions for login.gov
-
-Globals:
-  Function:
-    Timeout: 60
-    MemorySize: 128
-    Runtime: ruby2.7
-    Environment:
-      Variables:
-        HTTP_PROXY: !Sub
-          - '{{resolve:ssm:/${Environment}/network/outboundproxy/url:1}}'
-          - Environment: !Ref environment
-        HTTPS_PROXY: !Sub
-          - '{{resolve:ssm:/${Environment}/network/outboundproxy/url:1}}'
-          - Environment: !Ref environment
-        NO_PROXY: !Sub
-          - '{{resolve:ssm:/${Environment}/network/outboundproxy/no_proxy:1}}'
-          - Environment: !Ref environment
-        ENVIRONMENT_NAME: !Ref environment
-    Tags:
-      environment: !Ref environment
-
-Resources:
-  ProofAddressFunction:
-    Type: AWS::Serverless::Function 
-    Properties:
```

Comment:
Observing that we're including `MemorySize: 128` and `VersionDescription: !Ref gitsha` properties in the new generated output that did not exist previously, which seems reasonable given the common configuration.